js/filter.js

```diff
@@ -48,4 +110,38 @@ export const filterToParams = (options = {}, nameAlias = {}, nameFunc = {}, filt
     return params;
 };
 
+const _buildFilter = (field, arithOp, value, keywordFields) => {
+    if (!value) return [`${field}:${arithOp}:${value}`];
+
+    // verifies if the value is a keyword, if not
+    // returns the filter query with the given value
+    const result = value.match(KEYWORD_REGEX);
+    if (!result || !Object.keys(KEYWORDS).includes(value)) return [`${field}:${arithOp}:${value}`];

```

Comment:
Can't this be simplified?

```suggestion
    const keywordF = KEYWORDS[value];
    if (!keywordF) return [`${field}:${arithOp}:${value}`];
```